### PR TITLE
Add slack notifications for build failures

### PR DESCRIPTION
**Pull request by LinqLover**

### CODE REVIEW

**Comment by LinqLover** [comment]
Not yet tested.

**Comment by LinqLover** [comment]
Now test. Should work!

File: .travis.yml [hunk 1]
```diff
@@ -43,3 +43,6 @@ jobs:
 notifications:
   email:
     if: type = cron
+  slack:
+    rooms: hpi-swa-teaching:bfvhBDSWLe3kuUnRXk2JmSLp#squeak
```
Comment (codeZeilen) [inline .travis.yml:47]: This the hash sensitive information? How does the authorization with Slack work?

Comment (LinqLover) [reply .travis.yml]: So sorry for missing such an obvious security concern! I have reset the old token and generated and encrypted a new one. All details can be found here: https://docs.travis-ci.com/user/notifications/#configuring-slack-notifications Unfortunately, notifications seem not to work properly when using encrypted, I tested it multiple times ...

Comment (codeZeilen) [reply .travis.yml]: No worries. Maybe the token has to be in quotes? The documentation suggests so, but I would have assumed that this does not make much of a difference. Or did travis add this directly to the config?

Comment (LinqLover) [reply .travis.yml]: The latter. Hmm, and email notifications did not work either ... Maybe we should really consider moving to GitHub Actions at some time. What do you think?

Comment (codeZeilen) [reply .travis.yml]: Yes, we should probably move soonish... I imagine that we would not need to rework too much of the infrastructure. We might even be able to use one of the latex actions we use for paper and thesis writing.

Comment (LinqLover) [reply .travis.yml]: Alright! Created #101 and added it to my to-do list :)

**Pull request closed without merging**